admin/tests/integration/components/routes/authenticated/target-profiles/list-items_test.js

```diff
@@ -66,4 +67,20 @@ module('Integration | Component | routes/authenticated/target-profiles | list-it
     assert.dom('[aria-label="Profil cible"]').containsText(123);
     assert.dom('[aria-label="Profil cible"]').containsText('Profile Cible 1');
   });
+
+  test('it should display target profile status as "Obsolète" with grey icon when target profile is outdated', async function(assert) {
```

Comment:
`it should display target profile status as "Obsolète" with icon when target profile is outdated`